MainModule/Server/Dependencies/Assets/Clipper.client.lua

```diff
@@ -1,14 +1,11 @@
-local hum = script.Parent
-local char = hum.Parent
-local torso = char:FindFirstChild("HumanoidRootPart")
-local origY = torso.Position.Y
+local Humanoid = script.Parent
+local Character = Humanoid.Parent
+
 game:GetService("RunService").Stepped:Connect(function()
-	for _,v in pairs(char:GetDescendants()) do
-		if v:IsA("BasePart") then
-			v.CanCollide = false
+	for _, Object in pairs(Character:GetDescendants()) do
+		if Object:IsA("BasePart") and Object.CanCollide then
+			Object.CanCollide = false
 		end
 	end
```

Comment:
Are you sure setting the collisions are necessary in the first place?

Another way to do this more performantly is use `NoCollisionConstraint`